qbx_consumables/server.lua

```diff
@@ -120,6 +144,34 @@ lib.callback.register('consumables:server:usedItem', function(source, item)
 end)
 
 -- Added for ox_inv until I make a proper qbx bridge
-RegisterNetEvent('consumables:server:addThirst', addThirst)
 
-RegisterNetEvent('consumables:server:addHunger', addHunger)
+---sets player thirst level
+---@deprecated use setThirst export instead
+---@param amount number new hunger level
+RegisterNetEvent('consumables:server:addThirst', function (amount)
+    setThirst(source, amount)
+end)
+
+---sets player hunger level
+---@deprecated use setHunger export instead
+---@param amount number new hunger level
+RegisterNetEvent('consumables:server:addHunger', function (amount)
+    setHunger(source, amount)
+end)
+
+---client-side call, sets player thirst level
+---@param amount number
+RegisterNetEvent('consumables:server:setThirst', function (amount)
+    if not GetInvokingResource() then setThirst(source, amount) end
+end)
+
+---client-side call, sets player hunger level
+---@param amount number
+RegisterNetEvent('consumables:server:setHunger', function (amount)
+    if not GetInvokingResource() then setHunger(source, amount) end
```

Comment:
Can you explain the purpose behind the `not GetInvokingResource()` conditional here for me?